### PR TITLE
minor style changes

### PR DESCRIPTION
**Pull request by CumpsD**
Similar to https://github.com/jet/FsCodec/pull/32 and https://github.com/jet/equinox/pull/191, a bunch of small style changes to make it more consistent

### CODE REVIEW

File: src/Propulsion.EventStore/Checkpoint.fs [hunk 1]
```diff
@@ -36,44 +37,48 @@ module Fold =
     type State = NotStarted | Running of Events.Snapshotted
 
     let initial : State = NotStarted
+
```
Comment (bartelink) [inline src/Propulsion.EventStore/Checkpoint.fs]: initial kinda belongs with evolve and fold (dotnet-templates and equinox do this pretty consistently)

File: src/Propulsion.EventStore/Checkpoint.fs [hunk 2]
```diff
@@ -36,44 +37,48 @@ module Fold =
...
     let private evolve _state = function
         | Events.Started { config = cfg; origin=originState } -> Running { config = cfg; state = originState }
         | Events.Updated e | Events.Checkpointed e | Events.Overrode e -> Running { config = e.config; state = e.pos }
         | Events.Snapshotted runningState -> Running runningState
+
     let fold : State -> Events.Event seq -> State = Seq.fold evolve
     let isOrigin _state = true // we can build a state from any of the events and/or an unfold
```
Comment (bartelink) [inline src/Propulsion.EventStore/Checkpoint.fs]: blank line after fold, not before

File: src/Propulsion/Parallel.fs [hunk 3]
```diff
@@ -50,12 +52,15 @@ module Scheduling =
             mutable remaining : int // number of outstanding completions; 0 => batch is eligible for completion
             mutable faults : ConcurrentStack<exn> // exceptions, order is not relevant and use is infrequent hence ConcurrentStack
             batch: Batch<'M> }
+
         member private __.RecordOk(duration : TimeSpan) =
             // need to record stats first as remaining = 0 is used as completion gate
             Interlocked.Add(&__.elapsedMs, int64 duration.TotalMilliseconds + 1L) |> ignore
             Interlocked.Decrement(&__.remaining) |> ignore
+
```
Comment (bartelink) [inline src/Propulsion/Parallel.fs]: leave alongside prev?

File: src/Propulsion/Streams.fs [hunk 4]
```diff
@@ -101,7 +104,7 @@ module StreamName =
     /// will necessarily adhere to the `{category}-{aggregateId}` form
     /// This helper ensures there's always a dash (even if that means having a defaultStringId)
     let parseWithDefaultCategory defaultCategory (rawStreamName : string) : StreamName =
-        if rawStreamName.IndexOf '-' = -1 then String.Concat(defaultCategory,"-",rawStreamName)
+        if rawStreamName.IndexOf '-' = -1 then String.Concat(defaultCategory, "-", rawStreamName)
```
Comment (bartelink) [inline src/Propulsion/Streams.fs]: wth I wrote that only last week :shame:

Comment (CumpsD) [reply src/Propulsion/Streams.fs]: 🥇

**Comment by CumpsD** [comment]
remarks done, yes it took a bit to read all of that code :) I still don't grasp everything of it, but as a first take at least I have some context now :)

**Comment by bartelink** [comment]
🙏Thanks once again for going to all this effort

**Pull request merged**